www/scipylib/faq.rst

```diff
@@ -238,29 +239,25 @@ matrix argument for solving `generalized eigenvalue problems`_.
 Python version support
 ----------------------
 
-Do NumPy and SciPy support Python 3.x?
-######################################
+Do NumPy and SciPytill support Python 2.7?
+##########################################
 
-NumPy and SciPy support the Python 2.x series, (versions 2.6 and 2.7),
-as well as Python 3.2 and newer. The first release of NumPy to
-support Python 3 was NumPy 1.5.0. Python 3 support in SciPy starts
-with version 0.9.0.
+The last version of NumPy to support Python 2.7 is numpy 1.16.x. The last SciPy version
+to do so is 1.2.x.
+The first release of NumPy to support Python 3.x was NumPy 1.5.0.
+Python 3 support in SciPy was introduced in 0.9.0.
 
-Does NumPy/SciPy work with Jython?
-##################################
+Does NumPy/SciPy work with Jython or C#/.NET?
+#############################################
 
-No. Simply put, Jython runs on top of the Java Virtual Machine and has no
-way to interface with extensions written in C for the standard Python
-(CPython) interpreter.
+No, neither are supported. Jython never worked, because it runs on top of the
+Java Virtual Machine and has no way to interface with extensions written in C
+for the standard Python (CPython) interpreter.
 
-Does NumPy/SciPy work with IronPython (.NET)?
-#############################################
+Some years ago there was an effort to make NumPy and SciPy compatible with .NET.
+Some users at the time reported success in using NumPy with `Ironclad
+<https://code.google.com/archive/p/ironclad>`_ on 32-bit Windows.
```

Comment:
Could we add a Does NumPy/SciPy work with PyPy? with the answer "In general, yes. Since much of NumPy and SciPy is implemented as c-extension modules, the code may not run any faster, but as always when benchmarking, your experience is the best guide."

Reply:
yes good question. I will tweak the answer a bit, because it now looks like it will be as fast or faster, but in general it's way slower and not yet recommended for production usage

Reply:
added a FAQ entry, please review. I didn't literally say "not yet ready for production" because that hopefully will change soon and we shouldn't discourage people too much, but some sort of disclaimer is needed.

Reply:
Looks appropriate for now, thanks